src/InputChildren.tsx

```diff
@@ -95,7 +86,7 @@ export class InputChildren extends React.Component<InputChildren.Props, State> {
   }
 
   componentDidUpdate() {
-    this.computeChildrenSize();
+    setTimeout(this.computeChildrenSize);
```

Comment:
following react guidelines: https://it.reactjs.org/docs/react-component.html
change state should be called immediately in `componentDidUpdate` and we should add a guard that checks if props are really changed

Reply:
the check is already there (in `computeChildrenSize`), but not enough apparently 🤷‍♂

Reply:
inside computeChildrenSize we are checking props against current state, not against previous props, right?

plus, I fear that setTimeout here could be dangerous because it postpones at next cycle the setState

Reply:
we are not checking props, we are reading directly from the DOM (`clientWidth` and `clientHeight`) and comparing against `state`

I'm not saying I'm a fan of `setTimeout`, but I don't have better ideas 😂 

One thing I can improve is guarding that setState to run only when the component is mounted. I just pushed a commit to do this